app/src/main/res/values-es/strings.xml

```diff
@@ -173,7 +173,7 @@ Inténtelo de nuevo más tarde, por favor.</string>
   <string name="np_desc_rt">sólo larga distancia</string>
   <string name="np_desc_rt_networks">Alianza Railteam</string>
   <string name="np_region_finland">Finlandia</string>
-  <string name="np_desc_hsl">Hélsinki</string>
+  <string name="np_desc_fi">Hélsinki, Turku, Tampere, Jyväskylä, Oulu</string>
```

Comment:
Thanks, but this will be overridden by Transifex next time I update anyway.

Reply:
Ah, good point, I completely forgot about Transifex. I'll remove this from the patch, as here isn't much of a point of having it here and just adds noie to the patch.